### PR TITLE
Portfolio: Added contact form and footer sections.

### PR DESCRIPTION
**Pull request by ernestmusong**
I have added the following sections to my porflio

- Contact Form
To collect data using [Formspree](https://formspree.io/html/)

- Buttons interactions (active and hover)

- Footer containing social links

### CODE REVIEW

**Comment by MaganaAsiati** [comment]
hello

File: index.html [hunk 1]
```diff
@@ -252,5 +252,53 @@ <h4>Languages</h4>
...
     <!-- end cards -->
+
+    <!-- Form section-->
+    <section class="section" style="margin-top: 60px;">
+        <div class="contact-text">
+            <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+        </div>
+        <!-- form -->
+        <div class="form-wrapper">
+            <form class="contact-form">
```
Comment (MaganaAsiati) [inline index.html:263]: Please make sure that you collect data by using the [Formspree service](https://formspree.io/html) according to the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md). Following is a quote for a related requirement :+1: 

212893407-26216f0b-835a-4a62-bdfd-870c1bcc1923


Collect data by using the [Formspree service](https://formspree.io/html).

 You can do this by following the guidelines from [here](https://formspree.io/html/) link. Kindly ensure that your action attribute is similar to the one mentioned below arrow_down_small. However, make sure that you use your personalized form id. Ensure also that the form uses the action and method attribute to integrate with [formspree service](https://formspree.io/html).
action value should be similar to https://formspree.io/f/{form_id}
method value equals POST

File: index.html [hunk 2]
```diff
@@ -252,5 +252,53 @@ <h4>Languages</h4>
...
+                <ul>
+                    <li><input type="text" id="name" name="full_name" maxlength="30" placeholder="Full name" required></li>
+                    <li><input type="email" id="mail" name="user_email" placeholder="Email address" required></li>
+                    <li><textarea id="msg" name="user_message" maxlength="500" placeholder="Message" required></textarea></li>
+                </ul>
+                <button type="submit" class="section-button" style="margin-top: 36px;">
+                    get in touch
+                </button>
```
Comment (MaganaAsiati) [inline index.html:271]: - Kindly add the button interactions such as hover, active (when it's pressed), disabled, and enable (default state) are styled as shown in the Figma template. :+1: 

image

**Review by MaganaAsiati — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️♻️♻️ 
## Hi @ernestmusong , I'm @MaganaAsiati, your reviewer for this PR  :wave: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close-gif-4

## Highlights

- Descriptive README file  :heavy_check_mark:👨🏻‍💻
- Great work using the right GitHub Flow.:heavy_check_mark:
- No Linter errors :heavy_check_mark:

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
- [ ] [OPTIONAL] Please make sure to make your commit messages more descriptive in the future. :heavy_check_mark:👨🏻‍💻
- [ ] kindly  Add a meaningful description to your pull request in the future which gives more information about what you implement in this milestone :heavy_check_mark:👨🏻‍💻


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by ernestmusong** [comment]
Hi @MaganaAsiati thank you for your feedback. I have now made all required changes, please check!

**Review by Lizdev-05 — APPROVED** [review]
Hi @ernestmusong, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
Descriptive README file ✔️👨🏻‍💻
Great work using the right GitHub Flow.✔️
No Linter errors ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**